recipes/openexr/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class OpenEXRConan(ConanFile):
+    name = "openexr"
+    description = "OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & " \
+                  "Magic for use in computer imaging applications."
+    license = "BSD"
+    homepage = "https://github.com/openexr/openexr"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "namespace_versioning": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "namespace_versioning=True", "fPIC=True"
+    generators = "cmake"
+    exports_sources = ["IlmImf__ImfSystemSpecific.cpp.patch"]
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+
+    def requirements(self):
+        self.requires('zlib/1.2.11')
+
+    def configure(self):
+        if "fPIC" in self.options.fields and self.options.shared:
```

Comment:
I'm not sure if you should force. I know that doesn't make sense building shared + non-fPIC, but it's a generic rule. If we accept this, we should update ALL recipes.